### PR TITLE
we did it

### PR DESCRIPTION
**Pull request by caseymoe**

### CODE REVIEW

File: SamplePlugin/Classes/Inventories/Crystals.cs [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace EasyInventoryManager.Classes.Inventories
 {
-    internal class Crystals
+    internal class Crystal
```
Comment (PinkishCow) [inline SamplePlugin/Classes/Inventories/Crystals.cs:9]: Why

File: SamplePlugin/Classes/Inventories/IInventory.cs [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace EasyInventoryManager.Classes.Inventories
 {
-    internal interface IInventory
+    internal class IInventory
```
Comment (PinkishCow) [inline SamplePlugin/Classes/Inventories/IInventory.cs:9]: No :(  The first I implies interface, if we want something different make new file

File: SamplePlugin/Classes/Inventories/Crystals.cs (hunk 1 above)
Comment (PinkishCow) [inline SamplePlugin/Classes/Inventories/Crystals.cs:9]:
```c#
internal class Crystals
```

**Review by PinkishCow — APPROVED** [review]

File: SamplePlugin/Classes/Inventories/Crystals.cs (hunk 1 above)
Comment (caseymoe) [reply SamplePlugin/Classes/Inventories/Crystals.cs:9]: i forgor 💀

**Pull request merged**